test/index.js

```diff
@@ -27,7 +37,8 @@ describe("depcheck", function () {
     var absolutePath = path.resolve("test/fake_modules/good_es6");
 
     depcheck(absolutePath, { "withoutDev": true }, function checked(unused) {
-      assert.equal(unused.dependencies.length, 0);
+      assert.equal(unused.dependencies.length, 1);
+      assert.equal(unused.dependencies[0], "name-import-as");
```

Comment:
I've gone ahead and added some comments explaining this in https://github.com/woodb/depcheck-es6/commit/027d978aee04db762c8efe8b3188940d85c31898